app/router/config-bridge/config-bridge.go

```diff
@@ -24,19 +24,21 @@ import (
 )
 
 var (
-	Route = "/config/bridge"
+	Route        = "/config/bridge"
+	BridgeConfig *parser.Bridge
```

Comment:
This always returns the exact same from the initial load from the local configuration

Reply:
Actually what's happening is it fetches the latest config from the topic, saves it, then you register the router and it overrides it.